File: CS/ExportOptionSample/SimpleDataSource.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ExportOptionSample {
    public class SimpleData {
        public int ID { get; set; }
        public string Name { get; set; }
        public int Age { get; set; }

        public List<SimpleData> GetData() {
            var rand = new Random();
            List<SimpleData> list = new List<SimpleData>();
            for (int i = 0; i < 100; i++)
                list.Add(new SimpleData() { ID = i, Name = "Name " + i, Age = rand.Next(18, 45) });
            return list;
        }
    }
}

File: VB/ExportOptionSample/SimpleDataSource.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks

Namespace ExportOptionSample
	Public Class SimpleData
		Public Property ID() As Integer
		Public Property Name() As String
		Public Property Age() As Integer

		Public Function GetData() As List(Of SimpleData)
			Dim rand = New Random()
			Dim list As New List(Of SimpleData)()
			For i As Integer = 0 To 99
				list.Add(New SimpleData() With {
					.ID = i,
					.Name = "Name " & i,
					.Age = rand.Next(18, 45)
				})
			Next i
			Return list
		End Function
	End Class
End Namespace
